frontend/src/components/regions/UsersAndGroups/utils.test.js

```diff
@@ -0,0 +1,81 @@
+import {
+  getPrStatus,
+  getTildeReviewStatus,
+  timeDifferenceInDays,
+  configuration,
+  statuses,
+} from "./utils";
+
+const {
+  PR_ERROR_AGE_THRESHOLD,
+  PR_WARNING_AGE_THRESHOLD,
+  TILDE_ERROR_AGE_THRESHOLD,
+  TILDE_WARNING_AGE_THRESHOLD,
+} = configuration;
+
+const { ERROR_STATUS, WARNING_STATUS, DEFAULT_STATUS } = statuses;
+
+test("getPrStatus function returns error string when the pr age is 2 days or older", () => {
+  const date = new Date();
+  const card = {
```

Comment:
No need to define a `card`.  Just use the date. getPrStatus doesn't care about cards, it only cares about dates. Can you please remove the unnecessary concepts. Keep it clean and streamlined.

The way you are calculating the date is correct though.

Just do this:

```
time = date.getDate() - PR_ERROR_AGE_THRESHOLD
 expect(time).toBe(ERROR_STATUS);
```